src/server/routes/defaultRoute.js

```diff
@@ -90,9 +90,8 @@ async function doRender(req) {
     ''
   );
 
-  const html = await renderToStringWithData(Page);
```

Comment:
jeg antar at det ikke var en god grunn til at vi rendret to ganger, men om noen vet grunnen så skrik ut :smile: